supernode/server/0.3_bridge.go

```diff
@@ -121,9 +121,11 @@ func (s *Server) pullPieceTask(ctx context.Context, rw http.ResponseWriter, req
 	if !cutil.IsEmptyStr(dstCID) {
 		dstDfgetTask, err := s.DfgetTaskMgr.Get(ctx, dstCID, taskID)
 		if err != nil {
-			return err
+			logrus.Warnf("failed to get dfget task by dstCID(%s) and taskID(%s), and the srcCID is %s, err: %v",
```

Comment:
I think it's not a good idea to ignore the error here. Instead, maybe we can delete the `dstCID` after migrate for `dfget`.

Reply:
I do not think so. Here, dstCID is used to obtain the corresponding peerID, and then the state of the corresponding peerID is updated. If there is no corresponding dfgetTask at suopernode, then there is no need to update the state of the peerID. So you can continue to execute. 
Of course, dfget can delete the dstCID and what's more, it need to delete dstCID for all items which are stored in the queue.

Reply:
OK. I agree with you. And I think we should speed ​​up the progress of HA. @yunfeiyangbuaa 😄